index.html

```diff
@@ -0,0 +1,206 @@
+              <a href="#">Join</a>
+            </li>
+            <li class="links">
+              <a href="#">Sponsor</a>
+            </li>
+            <li class="links">
+              <a href="#">News</a>
+            </li>
+          </ul>
+          <div class="company">
+            <a href="index.html">Securam Circle</a>
+          </div>
+        </nav>
+    </div>
+  </header>
+
+<section class="home-page">
+  <div class="title">
+    <h1 class="intro-headline">"Hello!! It's Afrobeat to the world"</h1>
+    <h1 class="main-headline" >NIGERIA MUSIC SUMMIT 2021</h1>
+    <div class="theme">
+      <p>
+        Celebration of Nigerian hiphop miusic, its global reception, and challenges. creating a positive change in Nigerian music industry.
+      </p>
+      </div>
+      <div class="date">
+       <h2> 2021.12.16(THU) ~ 17(FRI) </h2>
+       <p>@ Eko Hotels & Suites <br> 1415 Adetokunbo Ademola Street, <br>Victoria Island Lagos </p>
+      </div>
+  </div>
+</section>
+
+<!-- About section -->
+
+<section class="program">
+  <div class="main-program">
+    <h3>Main Program</h3>
+    <hr>
+  </div>
+  <div class="program-list">
+
+    <div class="lecture">
+    <div class="icons">
+      <img src="image/program1.png" alt="">
+    </div>
+      <h3>Lecture</h3>
+    <div class="content"><p>
+    Listen to guest lecturer from diffrent aspect of the music<br> industry shearing their thoughts.
+    </p>
+    </div>
+    </div>
+  
+    <div class="lecture">
+    <div class="icons">
+      <img src="image/program2.png" alt="">
+    </div>
+      <h3>MUSIC</h3>
+    <div class="content"><p>
+    Listen to guest lecturer from diffrent aspect of the music<br> industry shearing their thoughts.
+    </p>
+    </div>
+    </div>
+  
+    <div class="lecture">
+    <div class="icons">
+      <img src="image/program3.png" alt="">
+    </div>
+      <h3>Forum</h3>
+    <div class="content"><p>
+    Listen to guest lecturer from diffrent aspect of the music<br> industry shearing their thoughts.
+    </p>
+    </div>
+    </div>
+
+    <div class="lecture">
+    <div class="icons">
+      <img src="image/program4.png" alt="">
+    </div>
+      <h3>Wrokshop</h3>
+    <div class="content"><p>
+    Listen to guest lecturer from diffrent aspect of the music<br> industry shearing their thoughts.
+    </p>
+    </div>
+    </div>
+
+    <div class="lecture">
+    <div class="icons">
+      <img src="image/program5.png" alt="">
+    </div>
+      <h3>Q&A</h3>
+    <div class="content"><p>
+    Listen to guest lecturer from diffrent aspect of the music<br> industry shearing their thoughts.
+    </p>
+    </div>
+    </div>
+
+
+    </div>
+
+  <button class="join" type="button"> Join the summit</button>
+</section>
+
+ <div class="section-speakers-heading" id="featured">
+      <h2>Featured Speakers</h2>
+      <hr />
+      <div class="featured-speakers" id="speakers"></div>
+    </div>
+
+<section class="partners-section" id="partners">
```

Comment:
- [ ] Please adjust the paddings/margins to match the given design

Yours:
![Screenshot from 2021-11-18 19-22-16](https://user-images.githubusercontent.com/62705947/142474321-28b29529-a8a8-4805-8684-7fc62f7ae6ce.png)


Original
![Screenshot from 2021-11-18 19-22-17](https://user-images.githubusercontent.com/62705947/142474406-ae18be4d-83bb-49d1-9ab7-dbbd2dac3666.png)

Reply:
- [ ] Also for mobile version this footer needs to be updated in order to match the original

Yours:


![Screenshot from 2021-11-18 19-27-07](https://user-images.githubusercontent.com/62705947/142475072-0a6cf9ab-6419-436b-ad5b-e2d45c4d8adf.png)

Original

![Screenshot from 2021-11-18 19-27-08](https://user-images.githubusercontent.com/62705947/142475097-2ad1077e-a7b3-4231-93c8-c0082c7f13d3.png)

Reply:
Well done